script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt

```diff
@@ -241,17 +250,17 @@ I can't breathe. I can't move my legs.
 [sha:5d31e40716451df2972dd76198041210e1cad568]{
 -- Page 240, Offset 40132.
 -- ちょうどアルクェイドの手の陰になって、あの竜巻からは守られるだろう。
-She's right on the other side of Arcueid's hand, so she'll be protected from this vortex.
+She's on the other side of Arcueid's hand, in the lee, so she should be safe from this vortex.
 }
 [sha:e104c3586fc33506390c6f54f68f83ecb1ba5ccf]{
 -- Page 240, Offset 40133.
 -- 俺はこのまま、自らの愚かさの代償を払うだけ。
-Right now, I'm the only one who will pay the price for my own foolishness.
+I'm the only one who will pay the price for my foolishness.
 }
 [sha:3ce23e25ac444a55f073c8c210279bb9be6cb71b]{
 -- Page 241, Offset 40134.
 -- ……死を受け入れた事による脱力か。
-...Am I exhausted because I've accepted death?
+I feel exhausted... is it because I've accepted my own death?
 }
 [sha:03e994c8e7718cca2c994a743fe0b5a53cd4ebb4]{
 -- Page 241, Offset 40135. Mods: @n.
```

Comment:
This is a sound of her blowing at him

>Phuwww―――

Or something? Not sure.

Reply:
Huff? Puff? Blow this house down?

Reply:
Puff works I guess